ocean_provider/utils/datatoken.py

```diff
@@ -3,13 +3,13 @@
 from datetime import datetime
 from typing import Optional
 
-from jsonsempai import magic  # noqa: F401
 from artifacts import ERC20Template
 from eth_keys import KeyAPI
 from eth_keys.backends import NativeECCBackend
 from eth_typing.encoding import HexStr
 from eth_typing.evm import HexAddress
 from hexbytes import HexBytes
+from jsonsempai import magic  # noqa: F401
```

Comment:
please take care: you have moved the jsonsempai after the artifacts imports.

Reply:
Done.